tests-e2e/cypress/e2e/edx-frontend-district/districtDetails.cy.ts

```diff
@@ -2,21 +2,28 @@ import selectors from '../../support/selectors';
 import { AppSetupData } from '../../../cypress.config';
 import { vInputParentOf } from 'tests-e2e/cypress/support/utils';
 
+before(() => {
+  cy.task<AppSetupData>('dataLoad').then(() => {
```

Comment:
Great question! Yesterday, I saw some flakiness in the tests without before. However, It seems reasonable to run it without the "before" block since we are using "beforeEach" already in our description. I re-ran the tests again right now. They seem to work fine now.